src/settings.ts

```diff
@@ -236,6 +237,15 @@ export class ConnectionSettingsService {
     return loadedConnection;
   }
 
+  async removeBindingsForConnection(connectionId: string) {
+    const bindingService = new BindingService(this.client, this);
```

Comment:
Initialisation is needed here (as opposed to passing the service to the constructor) to "avoid" circular dependency between the two services